src/components/members.component.js

```diff
@@ -29,45 +23,90 @@ type Props = {
 
 const avatarSize = 30;
 
-const styles = StyleSheet.create({
-  container: {
-    marginTop: 30,
-  },
-  avatarContainer: {
-    backgroundColor: colors.greyLight,
-    borderRadius: avatarSize / 2,
-    width: avatarSize,
-    height: avatarSize,
-  },
-  avatar: {
-    borderRadius: avatarSize / 2,
-    height: avatarSize,
-    width: avatarSize,
-  },
-  list: {
-    marginTop: 0,
-  },
-  sectionTitle: {
-    color: colors.black,
-    ...fonts.fontPrimaryBold,
-    marginBottom: 10,
-    paddingLeft: 15,
-  },
-  sectionTitleSmall: {
-    color: colors.primaryDark,
-    ...fonts.fontPrimarySemiBold,
-    marginBottom: 10,
-    paddingLeft: 15,
-  },
-  flatList: {
-    paddingHorizontal: 15,
-  },
-  scrollGradient: {
-    position: 'absolute',
-    width: 15,
-    height: avatarSize,
-  },
-});
+const Container = styled.View`
+  margin-top: 30;
+  ${props => props.containerStyle};
+`;
+
+const AvatarContainer = styled.TouchableHighlight`
+  background-color: ${colors.greyLight};
+  border-radius: ${avatarSize / 2};
+  width: ${avatarSize};
+  height: ${avatarSize};
+  ${props => props.style};
+`;
+
+const Avatar = styled.Image`
+  border-radius: ${avatarSize / 2};
+  height: ${avatarSize};
+  width: ${avatarSize};
+`;
+
+type StyledListProps = {
+  style: Object,
+  children: Element,
+};
+
+const StyledList = ({ style, children }: StyledListProps) => (
+  <List containerStyle={style}>{children}</List>
+);
+const NoMembersList = styled(StyledList)`
+  margin-top: 0;
+`;
+
+const SectionTitle = styled.Text`
+  color: ${props => (props.isTitleSmall ? colors.primaryDark : colors.black)};
+  ${props =>
+    props.isTitleSmall ? fonts.fontPrimarySemiBold : fonts.fontPrimaryBold};
+  margin-bottom: 10;
+  padding-left: 15;
+`;
+
+type StyledFlatListProps = {
+  style: Object,
+  data: Array,
+  renderItem: Function,
+  keyExtractor: Number,
+};
+
+const StyledFlatList = ({
+  style,
+  data,
+  renderItem,
+  keyExtractor,
+}: StyledFlatListProps) => (
+  <FlatList
+    contentContainerStyle={style}
+    data={data}
+    showsHorizontalScrollIndicator={false}
+    renderItem={renderItem}
+    keyExtractor={keyExtractor}
+    horizontal
+  />
+);
+const MembersFlatList = styled(StyledFlatList)`
+  padding-left: 15;
+  padding-right: 15;
+`;
+
+type GradientProps = {
+  style: Object,
+  gradientColors: Array,
+};
+const Gradient = ({ style, gradientColors }: GradientProps) => (
+  <LinearGradient
+    style={style}
+    colors={gradientColors}
+    start={{ x: 0, y: 0.5 }}
+    end={{ x: 1, y: 0.5 }}
+  />
+);
+const ScrollGradient = styled(Gradient)`
+  position: absolute;
+  width: 15;
+  height: ${avatarSize};
+  ${props => props.gradientPosition};
+`;
```

Comment:
Some of these have new lines between them, others do not. Is there a pattern you're following or can we clean this up a little?

Reply:
@andrewda added lines between blocks